docs/configure.rst

```diff
@@ -0,0 +1,96 @@
+onfiguration Files
+=====================
+
+.. contents:: Table of Contents
+   :local:
+
+This is another new section that will talk about what can be configured and will explain the interaction between the renewal configuration file and the global configuration file. The text about the global configuration file has been moved here from using.rst
+
+.. _renewal-config-file:
+
+
+Renewal Configuration File
+
+Modifying the Renewal Configuration File
+----------------------------------------
+
+When a certificate is issued, by default Certbot creates a renewal configuration file that
+tracks the options that were selected when Certbot was run. This allows Certbot
+to use those same options again when it comes time for renewal. These renewal
+configuration files are located at ``/etc/letsencrypt/renewal/CERTNAME``.
+
+For advanced certificate management tasks, it is possible to manually modify the certificate's
+renewal configuration file, but this is discouraged since it can easily break Certbot's
+ability to renew your certificates. If you choose to modify the renewal configuration file
+we advise you to test its validity with the ``certbot renew --dry-run`` command.
+
+.. warning:: Modifying any files in ``/etc/letsencrypt`` can damage them so Certbot can no longer properly manage its certificates, and we do not recommend doing so.
+
+For most tasks, it is safest to limit yourself to pointing symlinks at the files there, or using
+``--deploy-hook`` to copy / make new files based upon those files, if your operational situation requires it
+(for instance, combining certificates and keys in different way, or having copies of things with different
+specific permissions that are demanded by other programs).
+
+If the contents of ``/etc/letsencrypt/archive/CERTNAME`` are moved to a new folder, first specify
+the new folder's name in the renewal configuration file, then run ``certbot update_symlinks`` to
+point the symlinks in ``/etc/letsencrypt/live/CERTNAME`` to the new folder.
+
+If you would like the live certificate files whose symlink location Certbot updates on each run to
+reside in a different location, first move them to that location, then specify the full path of
+each of the four files in the renewal configuration file. Since the symlinks are relative links,
+you must follow this with an invocation of ``certbot update_symlinks``.
+
+For example, say that a certificate's renewal configuration file previously contained the following
+directives::
+
+  archive_dir = /etc/letsencrypt/archive/example.com
```

Comment:
the builder gave this error: `configure.rst:46: WARNING: Literal block expected; none found.`
Which I think means you need to do something else to make this look like code
here's the documentation: http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#literal-blocks
Though I can't actually tell what's wrong